utils/data-file.c

```diff
@@ -663,13 +663,13 @@ static int create_data(struct uftrace_opts *opts)
 	write_dlopen_info(opts->dirname, &dlopen_msg, "libnothing.so");
 
 	/* create_session() requires a map file even if it's empty */
-	asprintf(&filename, "%s/sid-%.*s.map",
+	(void)asprintf(&filename, "%s/sid-%.*s.map",
```

Comment:
I think it's better to check it with one of test macro like `TEST_GE(asprint(...), 0)` to verify it returns 0 or greater.